pkg/operator/ceph/file/filesystem.go

```diff
@@ -116,3 +118,162 @@ func validateFilesystem(context *clusterd.Context, f cephv1.CephFilesystem) erro
 
 	return nil
 }
+
+const (
```

Comment:
want to put the consts at the top of the file?

Reply:
this additions just moved from another file that was deleted under /daemon, right?

Reply:
That's correct. The intent here was to follow the guideline of defining things as close to their use as possible, especially since they aren't likely to change. But I'm not sure if that is actually helpful here? Shall I move them to the top of the file?

Reply:
I'll move them to the top since I want to make some changes anyway. I think you're probably right.